website/static/downloads/swagger.yaml

```diff
@@ -895,6 +895,8 @@ components:
           properties:
             /:
               $ref: '#/components/schemas/cid'
+    pubKey:
+      $ref: '#/components/schemas/pubKey'
```

Comment:
this does not look right. pubkey is declared as reference to itself?

Reply:
yes, you are right. did it accidentally :D